R/measures.R

```diff
@@ -90,14 +90,39 @@ timeboth = makeMeasure(id = "timeboth", minimize = TRUE, best = 0, worst = Inf,
 #' @rdname measures
 #' @format none
 sse = makeMeasure(id = "sse", minimize = TRUE, best = 0, worst = Inf,
-  properties = c("regr", "req.pred", "req.truth"),
+  properties = c("regr", "fcregr", "req.pred", "req.truth"),
   name = "Sum of squared errors",
   note = "Defined as: sum((response - truth)^2)",
   fun = function(task, model, pred, feats, extra.args) {
     measureSSE(pred$data$truth, pred$data$response)
   }
 )
 
+#' @export mase
+#' @rdname measures
+#' @usage  none
+#' @format none
+mase = makeMeasure(
+  id = "mase",
+  minimize = TRUE,
+  name = "Mean Absolute Scaled Error",
+  properties = c("regr", "fcregr", "req.pred", "req.truth", "req.task"),
+  best = 0,
+  worst = Inf,
+  fun = function(task, model, pred, feats, extra.args) {
```

Comment:
Would it be possible to externalize that in `measureMASE` like we did it for the other measures?